### ISSUE TITLE
Enhancement: NC 17 support :)

### ISSUE THREAD

**Issue by Gatak** [ariselseng/camerarawpreviews#35]
Just a ping to ask for NC 17 compatibility :) Thanks!

**Comment by e-alfred** [comment]
@ariselseng It seems like you forgot to upload the release file for the 0.7.1 release?

**Comment by Schmuuu** [comment]
Yes, when trying to update or download the app, there is an error:
```
Client error: `GET https://github.com/ariselseng/camerarawpreviews/releases/download/v0.7.1/camerarawpreviews_nextcloud.tar.gz` resulted in a `404 Not Found` response: Not Found
```

**Comment by Aurnytoraink** [comment]
@Schmuuu Yeah me too, I think @ariselseng forgot to provide the final compiled file into his release

**Comment by paulcalabro** [comment]
I'm getting a 404 as well.

**Comment by ariselseng** [comment]
I found a bug in the release and could not remove the update before I went to work, so I removed the asset. I will push a new update as soon as I can.

**Comment by ariselseng** [comment]
Sorry for today guys. Usually I do not make a new release right before work. There was a bug that broke the preview generation and the worse thing is that the result is stored forever so I just needed to remove the asset from github to stop people from getting it. Eventually I figured out how to remove the release from apps.nextcloud.com as well (rest api).

A new release is up now that fixes the broken release. Have a nice weekend guys.
